playbook/app/pb_kits/playbook/playbook-rails.js

```diff
@@ -9,6 +9,10 @@ window.datePickerHelper = datePickerHelper
 import dialogHelper from './pb_dialog/dialogHelper'
 window.dialogHelper = dialogHelper
 
+// CopyContent
+import copyContent from './pb_docs/copyCode'
```

Comment:
This is used outside of docs. Please move

Reply:
this came from the [pr 2470](https://github.com/powerhome/playbook/pull/2470/files) that I merged to start working. I marged master to my branch and deleted this code.